tests/h/traversal/roots_test.py

```diff
@@ -76,24 +79,85 @@ def links_service(self, pyramid_config):
         return service
 
 
-class TestAuthClientResourceFactory(object):
-    def test_get_item_returns_an_authclient(self, pyramid_request):
-        authclient = AuthClient(name='test', authority='example.com')
-        pyramid_request.db.add(authclient)
-        pyramid_request.db.flush()
+class TestAuthClientRoot(object):
+    def test_getitem_returns_the_right_AuthClient(self, db_session, pyramid_request):
```

Comment:
Absolutely trivial, but might you consider replacing `right` in this method name with `correct`? Because of a background dealing with (hey, Zope, actually) DB relations that have a `left` and a `right` relation, I parse this method name (initially) as being about a right-side relation. In any case, _right_ can be a little ambiguous.